parmed/openmm/parameters.py

```diff
@@ -101,10 +106,18 @@ def from_parameterset(cls, params, copy=False):
         new_params.parametersets = params.parametersets
         new_params._combining_rule = params.combining_rule
         new_params.residues = params.residues
+        # Make sure we have a complete set of atom types in the _tuple list
+        if len(params.atom_types_tuple) == 0:
+            i = 1
+            for name, type in iteritems(params.atom_types):
```

Comment:
I think we want to use the convention that OpenMM `type` is the AMBER atom type, while the OpenMM `class` is the AMBER Lennard-Jones reduced atom class (where several types map to one typename for the purposes of assigning LJ parameters).